resources/assets/js/components/FindSkillsModal.tsx

```diff
@@ -0,0 +1,481 @@
+                    (childSkillCategory) =>
+                      childSkillCategory.depth === 2 &&
+                      childSkillCategory.parent_id === id,
+                  );
+                  return (
+                    <li
+                      key={id}
+                      data-h2-bg-color={`b(gray-1, ${
+                        expandedAccordions.includes(key) ? ".5" : "0"
+                      })`}
+                      data-h2-padding="b(tb, 1)"
+                      data-h2-border={`${
+                        index + 1 !== parentSkillCategories.length
+                          ? "b(gray-2, bottom, solid, thin)"
+                          : ""
+                      }`}
+                      data-h2-margin="b(tb, 0)"
+                    >
+                      <Accordion triggerPos="left">
+                        <Accordion.Btn
+                          buttonStyling=""
+                          type="button"
+                          addIcon={
+                            <i
+                              data-h2-font-weight="b(700)"
+                              className="fas fa-plus"
+                            />
+                          }
+                          removeIcon={
+                            <i
+                              data-h2-font-weight="b(700)"
+                              className="fas fa-minus"
+                            />
+                          }
+                          onClick={() =>
+                            setExpandedAccordions(
+                              expandedAccordions.includes(key)
+                                ? expandedAccordions.filter(
+                                    (accordionKey) => accordionKey !== key,
+                                  )
+                                : [...expandedAccordions, key],
+                            )
+                          }
+                        >
+                          <p data-h2-font-weight="b(700)">{name[locale]}</p>
+                        </Accordion.Btn>
+                        <p
+                          data-h2-padding="b(top, .25) b(bottom, 1) b(right, .5)"
+                          data-h2-font-color="b(black)"
+                          data-h2-font-size="b(small)"
+                          style={{ paddingLeft: "5rem" }}
+                        >
+                          {description[locale]}
+                        </p>
+                        <Accordion.Content>
+                          <ul
+                            data-h2-padding="b(all, 0)"
+                            className="no-list-style-type"
+                          >
+                            {childrenSkillCategories.map(
+                              (childSkillCatergory) => {
+                                return (
+                                  <li key={childSkillCatergory.key}>
+                                    <div
+                                      data-h2-dialog-actions
```

Comment:
This comment isn't a critique, but I'm just wondering what this `data-h2-dialog-actions` attribute does. Is it just for styling?

Reply:
Nope, no special styling. I think I was just copy/pasting code and forgot to remove that.